assets/css/footer.css

```
@@ -0,0 +1,245 @@
+#sun-footer {
+    align-items: center;
+    background-color: #f7f7f7;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+}
+
+.wrapper__footer {
+    max-width: 1140px;
+    width: 100%;
+}
+
+.wrapper__footer:first-of-type {
+    margin-top: 60px;
+}
+
+.about__section--first {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+}
+
+.about__section--first>div {
```

```suggestion
.about__section--first > div {
```
E fix tương tự những chỗ khác nữa nha